airflow-core/src/airflow/models/deadline.py

```diff
@@ -64,6 +70,9 @@ class Deadline(Base):
 
     dagrun = relationship("DagRun", back_populates="deadlines")
 
+    trigger_id = Column(Integer, ForeignKey("trigger.id"), nullable=True)
```

Comment:
I liked the pattern above of including a comment that explains what this is and/or how it's used.

Reply:
I agree, fixed!